src/tokens.md

```diff
@@ -153,16 +153,21 @@ which must be _escaped_ by a preceding `U+005C` character (`\`).
 Line-breaks are allowed in string literals. A line-break is either a newline
 (`U+000A`) or a pair of carriage return and newline (`U+000D`, `U+000A`). Both
 byte sequences are normally translated to `U+000A`, but as a special exception,
-when an unescaped `U+005C` character (`\`) occurs immediately before the
-line-break, then the `U+005C` character, the line-break, and all whitespace at the
-beginning of the next line are ignored. Thus `a` and `b` are equal:
+when an unescaped `U+005C` character (`\`) occurs immediately before a `\n`
+(`U+000A`), then the `U+005C` character, and all immediately following
```

Comment:
I think this should retain the "line-break" terminology, because it can be followed by `\r\n` too.  The issue is that crlf normalization has not made it into the reference, yet (see #626).

Reply:
Good call. Should be fixed.